docs: rename option of ResponseTime (#245)
tier4/CARET_doc: docs/visualization/path/response_time.md

    However, both will be referred to as "response time" for simplicity in this document.
<prettier-ignore-end>

> Three cases of response time is defined; `all`, `best`, `worst`, and `worst-with-external-latency`. With `all` case, CARET samples all time elapsed in a targeted path. With `best` case, CARET samples shortest time elapsed in a targeted path. With `worst` case, CARET samples largest time elapsed in a targeted path. With `worst-with-external-latency` case, the previous message input timing is taken into account. `best-to-worst` case includes most of all cases between `best` and `worst`.

Reviewer: ```suggestion
As a visualisation of the graph, four different cases can be specified in the `case` argument. Default value is 'all'; 'all', 'best', 'worst', and 'worst-with-external-latency'.

- `all` case
  - CARET displays the elapsed time from all input times in the same cycle.
- `best` case
  - CARET displays the elapsed time that is the shortest of all input times in the same cycle.
- `worst` case
  - CARET displays the elapsed time that is the longest of all input times in the same cycle. 
- 'worst-with-external-latency` case
  - CARET displays the elapsed time from the last input time of a previous cycle.
```

Author: Fixed it in [2211093](https://github.com/tier4/CARET_doc/pull/245/commits/2211093ddd6a8c037203df4f4072a5374e4bc7ab).

> This section shows two methods to visualize response time, `Histogram`, `Stacked Bar` and `TimeSeries`.

Reviewer: ```suggestion
This section shows three methods to visualize response time, `Histogram`, `Stacked Bar` and `TimeSeries`.
```

Author: Fixed it in [2211093](https://github.com/tier4/CARET_doc/pull/245/commits/2211093ddd6a8c037203df4f4072a5374e4bc7ab).

    The `output_notebook()` is needed to show figures inside Jupyter Notebook.
<prettier-ignore-end>

![response_time_histogram_all](../imgs/response_time_histogram_all.png)

```python
# plot best case
plot = Plot.create_response_time_histogram_plot(path, case='best')
plot.show()
```

![response_time_histogram_best](../imgs/response_time_histogram_best.png)

```python
# plot worst case
plot = Plot.create_response_time_histogram_plot(path, case='worst')
plot.show()
```

![response_time_histogram_worst](../imgs/response_time_histogram_worst.png)

```python
# plot worst-with-external-latency case
plot = Plot.create_response_time_histogram_plot(path, case='worst-with-external-latency')
plot.show()
```

![response_time_histogram_worst_with_external_latency](../imgs/response_time_histogram_worst_with_external_latency.png)

## Stacked Bar

The following scripts generate stacked bar graphs of response time.

> Horizontal axis of the stacked bar graph means timeseries and vertical axis means `Response time` in the all, best, worst, worst-with-external-latency case.

Reviewer: In the sample graph, `latency` is displayed, but is it the same value as response time?
```suggestion
Horizontal axis of the stacked bar graph means `system time` or `index` and vertical axis means response time.
```

Author: Fixed it in [2211093](https://github.com/tier4/CARET_doc/pull/245/commits/2211093ddd6a8c037203df4f4072a5374e4bc7ab).

Is the description of the vertical axis of the stacked bar correct as per the change?

# plot all case
plot = Plot.create_response_time_stacked_bar_plot(path)
plot.show()
# or
# plot = Plot.create_response_time_stacked_bar_plot(path, case='all')
# plot.show()
```

![response_time_stacked_bar_all](../imgs/response_time_stacked_bar_all.png)

```python
# plot best case
plot = Plot.create_response_time_stacked_bar_plot(path, case='best')
plot.show()
```

![response_time_stacked_bar_best](../imgs/response_time_stacked_bar_best.png)

```python
# plot worst case
plot = Plot.create_response_time_stacked_bar_plot(path, case='worst')
plot.show()
```

![response_time_stacked_bar_worst](../imgs/response_time_stacked_bar_worst.png)

```python
# plot worst-with-external-latency case
plot = Plot.create_response_time_stacked_bar_plot(path, case='worst-with-external-latency')
plot.show()
```

![response_time_stacked_bar_worst_with_external_latency](../imgs/response_time_stacked_bar_worst_with_external_latency.png)

The horizontal axis can be changed to `system time` or `index` by changing `plot.show()` to `plot.show(xaxis_type='index')`.

`[worst - best] response time` in legends refers to the difference in response time between the worst case and the best case.

## TimeSeries

> The following scripts generate timeseries graphs of response time. The horizontal axis means time, labeled as Time [s]. The vertical axis means `Response time` in the all, best, worst, worst-with-external-latency case.

Reviewer: ```suggestion
The following scripts generate timeseries graphs of response time. The horizontal axis means `system time` or `index`. The vertical axis means `Response time`.
```

Author: Fixed it in [2211093](https://github.com/tier4/CARET_doc/pull/245/commits/2211093ddd6a8c037203df4f4072a5374e4bc7ab).
